docs/nodes/gcp.md

```diff
@@ -8,6 +8,7 @@ Node classes list of gcp provider.
 ## gcp.analytics
 
 - **diagrams.gcp.analytics.Bigquery**, **BigQuery** (alias)
+- **diagrams.gcp.analytics.CloudTask**
```

Comment:
This has been implemented in diagrams.gcp.devtools.Tasks.

Reply:
Hi ryanyuan, Can we change the name space, devtools is quite misleading. I also tried to locate a separate namespace  for messaging (pub sub, cloud tasks), only found analytics. Any thoughts ?

Reply:
If you look at [https://cloud.google.com/products#section-9](https://cloud.google.com/products#section-9), Cloud Task is actually under dev tool category. Plus if you could open the GCP console, you can find the same categorisation on the navigation menu. And to be honest, Cloud Task is not for analytics. 

The namespace under gcp are based on the official categorisations from the link above.

Reply:
As @ryanyuan said, Cloud Tasks is under the dev tool category officially. Change the namespace is not a good option I think.